frontend/src/components/Forms/TeamEventCreditsForm/TeamEventsCreditDasboard.tsx

```diff
@@ -1,39 +1,30 @@
 import React, { useEffect, useState } from 'react';
 import { Form, Card, Message } from 'semantic-ui-react';
-import { useSelf } from '../../Common/FirestoreDataProvider';
+import { TeamEventsAPI } from '../../../API/TeamEventsAPI';
 import styles from './TeamEventCreditsForm.module.css';
 
-const TeamEventCreditDashboard = (props: { teamEvents: TeamEvent[] }): JSX.Element => {
+const REQUIRED_TEC_CREDITS = 3; // number of required tec credits in a semester
+
```

Comment:
This is a good idea in case they want to change the number from year to year. Could you use this const in the instructions too where it says " Every DTI member must complete 3 team event credits..." just so it keeps in sync with this number?